openlibrary/plugins/openlibrary/js/covers_add_manage.js

```diff
@@ -8,10 +11,6 @@ export function initAddCoverImport () {
         event.preventDefault();
     }
 
-    // page may not be loaded via iframe
```

Comment:
I'm not 100% sure about this following the last review, so will need to test this some more. Thanks for explaining.

Reply:
In this context `parent` would refer to the parent iframe window and thus closeThrobber would refer to a global method on the window object. No such closeThrobber method seems to exist so I feel safe removing this. Will mention it in the commit message